### ISSUE TITLE
Export directives with basic API

### ISSUE THREAD

**Issue by arturhun** [nogorilla/angular-cc-library#45]
Please export the directives so that user is able to use a template variable for more advanced UI stuff, like `ngIf`ing certain elements based on whether/how the card has been identified.

Also, provide public api to access information, such as what the card has been identified as.

```html
<input type="tel"
          id="input-cc-number"
          autocomplete="input-cc-number"
          formControlName="cardNumber"
          ccNumber
          #ccNumberInput="ccNumber"
/>
```

somewhere else:

```html
<p *ngIf="ccNumberInput.verifiedAs === 'visa'">
  show some info
</p>
```

or change CVC label to CID in case American Express

**Comment by thekip** [comment]
There is already a way to implement this, just use CreditCard class from a library, but I will also put this into a projects roadmap because this seems not really hard to do and will not break anything.

**Comment by thekip** [comment]
Implemented in [v2.0.0](https://github.com/nogorilla/angular-cc-library/releases/tag/v2.0.0)
